symfony/plugins/orangehrmMaintenancePlugin/lib/form/PurgeForm.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: administrator
+ * Date: 27/8/18
+ * Time: 6:04 PM
+ */
+class PurgeForm extends sfForm {
+    public function configure() {
+        $this->setWidgets($this->getWidgetList());
+        $this->setValidators($this->getValidatorList());
+        $this->getWidgetSchema()->setLabels($this->getLabelList());
+    }
+    public function getWidgetList() {
+        $widgets = array();
+        $widgets['employee'] = new ohrmWidgetEmployeeNameAutoFill(array('jsonList' => $this->getEmployeeListAsJson()));
+//        $widgets['check_employee_data'] = new sfWidgetFormInputHidden();
+
+        return $widgets;
+    }
+    public function getValidatorList() {
+        $validators = array();
+        $validators['employee'] = new ohrmValidatorEmployeeNameAutoFill(array('required' => true));
+        return $validators;
+    }
+    public function getLabelList() {
+        $requiredMarker = ' <em>*</em>';
+        $lableList = array();
+        $lableList['employee'] = __('Select Terminated Employee') . $requiredMarker;
+        return $lableList;
+    }
+
+
+
+    protected function getEmployeeListAsJson() {
+        $jsonArray = array();
+        $properties = array("empNumber", "firstName", "middleName", "lastName", 'termination_id','purged_at');
```

Comment:
why two standards here? camelcase and underscore

Reply:
Is it copy code from some base class ?

Reply:
yes. I get it from code base and modified that. I corrected that.

Reply:
I have to look into that more.